src/transports/https.js

```diff
@@ -5,13 +5,14 @@ import { Writable } from 'stream';
 
 const HOSTNAME = 'api.timber.io';
 const PATH = '/frames';
-const CONTENT_TYPE = 'application/msgpack';
+const CONTENT_TYPE = 'application/json';
```

Comment:
We need to add support for this, currently we only support `application/msgpack` and `application/logplex-1`. Created an issue.